src/main/java/it/gov/pagopa/pu/classification/repository/AssessmentsDetailRepository.java

```diff
@@ -3,9 +3,13 @@
 import it.gov.pagopa.pu.classification.model.AssessmentsDetail;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.rest.core.annotation.RepositoryRestResource;
+import org.springframework.data.rest.core.annotation.RestResource;
 
 
 @RepositoryRestResource(path = "assessments-details")
 public interface AssessmentsDetailRepository extends JpaRepository<AssessmentsDetail,Long> {
 
+  @RestResource(exported = false)
+  AssessmentsDetail getByAssessmentIdAndIuvAndIudAndOfficeCodeAndSectionCodeAndAssessmentCode(Long assessmentId, String iuv, String iud,String officeCode, String sectionCode, String assessmentCode);
```

Comment:
I prefer `find` as keywork on Repository
```suggestion
  AssessmentsDetail findByAssessmentIdAndIuvAndIudAndOfficeCodeAndSectionCodeAndAssessmentCode(Long assessmentId, String iuv, String iud,String officeCode, String sectionCode, String assessmentCode);
```